ibis/backends/pandas/execution/window.py

```diff
@@ -113,25 +105,18 @@ def get_aggcontext(
 
 @get_aggcontext.register(win.Window)
 def get_aggcontext_window(
-    window,
-    *,
-    scope,
-    operand,
-    operand_dtype,
-    parent,
-    group_by,
-    order_by,
-    **kwargs,
+    window, *, scope, operand, parent, group_by, order_by, **kwargs,
 ) -> AggregationContext:
     # no order by or group by: default summarization aggcontext
     #
     # if we're reducing and we have an order by expression then we need to
     # expand or roll.
     #
     # otherwise we're transforming
+    output_type = operand.type()
 
     if not group_by and not order_by:
-        aggcontext = agg_ctx.Summarize()
+        aggcontext = agg_ctx.Summarize(parent=parent, output_type=output_type)
```

Comment:
is output_type now *required* (wouldn't object if it was)

Reply:
It is not. It probably should be but currently this is created here: https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/core.py#L207 and the expr here can be a `TableExpr` so it's not clear what the output type even mean in this case..